python/ql/src/experimental/Security/CWE-348/ClientSuppliedIpUsedInSecurityCheckLib.qll

```diff
@@ -1,7 +1,6 @@
 private import python
 private import semmle.python.Concepts
 private import semmle.python.ApiGraphs
-private import semmle.python.dataflow.new.DataFlow
```

Comment:
This is the only one that surprises me. It is imported by `ApiGraphs` which should perhaps have done so privately..

Reply:
I remember that I tried to make the import of `DataFlow` private inside `ApiGraphs` when I did def-nodes for Python.  
But that caused a surprising amount of compile-errors, so I dropped that change.  

You could look into a better fix in a follow-up PR (like the Java team is doing now).

Reply:
Then it is perhaps best to revert just the change to this file. Then we will have an alert to remind us to solve it later.